modules/dhcp/providers/server/isc.rb

```diff
@@ -288,8 +298,7 @@ def read_config file, ignore_includes=false
       logger.debug "Reading config file #{file}"
       config = []
       File.readlines(file).each do |line|
-
-        line.strip! # remove left and right whitespace
+        line = line.split('#').first.strip # remove comments, left and right whitespace
```

Comment:
Why this change? The next line used to skip over comments (and isn't needed with this change).